### PR TITLE
BOM-2344 : OEP-18 and make upgrade

### PR DESCRIPTION
**Pull request by mzulqarnain1**
Relevant JIRA: https://openedx.atlassian.net/browse/BOM-2344

### CODE REVIEW

File: .github/workflows/requirements-upgrade.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Upgrade Python Requirements
```
Comment (jmbowman) [inline .github/workflows/requirements-upgrade.yml:1]: Do we still need this, with the new org-wide workflow template?

**Review by jmbowman — COMMENTED** [review]
We'll need to update the Read the Docs project configuration to use the new requirements file once this is merged.

File: .github/workflows/requirements-upgrade.yml (hunk 1 above)
Comment (mzulqarnain1) [reply .github/workflows/requirements-upgrade.yml]: Removed.

**Comment by mzulqarnain1** [comment]
> We'll need to update the Read the Docs project configuration to use the new requirements file once this is merged.

Sure. Can you guide me to where those configurations are located?

File: Makefile [hunk 2]
```diff
@@ -227,3 +227,21 @@ pseudoxml:
...
+	# Make sure to compile files after any other files they include!
+	$(PIP_COMPILE) -o requirements/pip_tools.txt requirements/pip_tools.in
+	$(PIP_COMPILE) -o requirements/base.txt requirements/base.in
+	$(PIP_COMPILE) -o requirements/dev.txt requirements/dev.in
+
+.PHONY: requirements
+requirements:
+	pip install -r requirements/base.txt
+
+dev_requirements:
```
Comment (feanil) [inline Makefile:246]: Not a blocker: Are we standardizing on underscore? I feel like most places use a dash instead.

Comment (mzulqarnain1) [reply Makefile]: I've changed this to hyphen instead.

**Pull request merged**